### PR TITLE
docs: improve README with more info

### PR DESCRIPTION
**Pull request by Nsttt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,50 @@
...
+
+```bash
+go build -o package-size-calculator
 ```
 
-## License
+## Contributing
 
-Package size calculator is licensed under the MIT License. See [LICENSE](LICENSE) for the full license text.
+Contributions are welcome! Please open an issue or submit a pull request for any improvements or bug fixes.
```
Comment (TheDevMinerTV) [inline README.md:50]: Please don't remove the License section, it should be there. Add the contributing section above instead

File: README.md [hunk 2]
```diff
@@ -1,26 +1,50 @@
...
+- [Docker Desktop for Mac](https://docs.docker.com/desktop/install/mac-install/)
+- [Docker Engine for Linux](https://docs.docker.com/desktop/install/linux-install/#generic-installation-steps)
+
+## Usage
+
+To calculate the size of an NPM package, run:
 
 ```bash
-package-size-calculator
+package-size-calculator <package-name>
```
Comment (TheDevMinerTV) [inline README.md:26]: The CLI doesn't accept a package name parameter.

File: main.go [hunk 3]
```diff
@@ -38,6 +38,11 @@ func main() {
 
 	flag.Parse()
 
+	if flag.NArg() == 0 {
+		flag.Usage()
+		return
+	}
```
Comment (TheDevMinerTV) [inline main.go:44]: The CLI doesn't accept a parameter, so this will always trigger and make it unusable

**Review by TheDevMinerTV — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Nsttt) [reply README.md]: Whoops. Done.

File: README.md (hunk 2 above)
Comment (Nsttt) [reply README.md]: Done.

File: main.go (hunk 3 above)
Comment (Nsttt) [reply main.go]: Nuked

**Review by TheDevMinerTV — APPROVED** [review]
LGTM

(though I'm going to squash the commits)

**Pull request merged**